2.4/root/usr/share/container-scripts/httpd/README.md

```diff
@@ -80,6 +80,8 @@ The Apache HTTP Server container image supports the following configuration vari
 **`HTTPD_LOG_TO_VOLUME`**  
        By default, httpd logs into standard output, so the logs are accessible by using the docker logs command. When `HTTPD_LOG_TO_VOLUME` is set, httpd logs into `/var/log/httpd24`, which can be mounted to host system using the container volumes. This option is only allowed when container is run as UID 0.
 
+**`HTTPD_MPM`**
+       The variable `HTTPD_LOG_TO_VOLUME` can be set to change the default Multi-Processing Module (MPM) from the package default MPM.
```

Comment:
Should this really be `HTTPD_LOG_TO_VOLUME`?

Reply:
Good catch, yes, thanks.